validator/api/src/main/java/tech/pegasys/artemis/validator/api/ValidatorTimingChannel.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.validator.api;
+
+import com.google.common.primitives.UnsignedLong;
+
+public interface ValidatorTimingChannel {
```

Comment:
Not sure yet what will actually drive this.  The validator service could potentially poll something like `/beacon/chain_head` to track where the beacon chain client is up to or maybe run it's own timer.  Might be better to have the beacon chain service publish these events though.

Possibly need too support both polling and having the beacon chain publish events so we are compatible with other clients.

Reply:
+1 to having both in order to be compatible with other clients. That being said, we should probably prioritize the beacon chain publishing events, since it's the more efficient approach.